packages/frint-props/src/withStore.js

```diff
@@ -8,6 +8,10 @@ import { scan } from 'rxjs/operators/scan';
 function makeStateProps$(store, mapper) {
   const state$ = from(store);
 
+  if (mapper === null) {
```

Comment:
I think if we pass `mapper` as `null`, there shouldn't be any subscription to the Store at all. since we are not expecting to read the state anywhere.

This is how it is in `master` now: https://github.com/frintjs/frint-props/blob/609b727b65528217321eb6ebe24f2c38293fa119/packages/frint-props/src/withStore.js#L8-L14

We can possibly update it like this:

```js
function makeStateProps$(store, mapper) {
  // when no mapper is given, we can just stream a plain object once, 
  // which will be merged with dispatchable action creators later
  if (!mapper) {
    return of({});
  }

  // continue with regular subscription otherwise
  const state$ = from(store);

  return state$.pipe(
    map(state => mapper(state)),
  );
}
```